financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ui/ConsentLogoHeader.kt

```diff
@@ -224,15 +221,23 @@ private fun AnimatedDotsWithFixedGradient(
 @Composable
 private fun Logo(imageBitmap: ImageBitmap) {
     val shape = RoundedCornerShape(18.dp)
-
-    Image(
-        bitmap = imageBitmap,
-        contentDescription = null,
+    Box(
         modifier = Modifier
             .size(LogoSize)
             .shadow(8.dp, shape)
             .clip(shape)
-    )
+    ) {
+        Crossfade(
```

Comment:
Nit: We can probably use `Crossfade` as the containing composable and remove the wrapping `Box`.

Even smaller nit: Let’s also add a label for the animation.

Reply:
Gotcha, then let’s keep the `Box`.